src/components/About.js

```diff
@@ -0,0 +1,17 @@
+import about_img_1 from "./assets/images/about_img_1.png";
+import about_img_2 from "./assets/images/about_img_2.png";
+import data from "./structure.json";
```

Comment:
Please call this data in the `App.js` file and send the required data as `props`.

Reply:
Done